### ISSUE TITLE
SNZB-01P Support Please

### ISSUE THREAD

**Issue by trhessel** [bwp91/homebridge-ewelink#520]
eWeLink BridgeeWeLinkWARNING[SNZB-01P] is not supported, do create a GitHub issue with the following info:
[7000] {"bindInfos":{},"subDevId":"af2adcfeff20ba847000","parentid":"1001dc6ebb","fwVersion":"2.0.0","battery":100,"trigTime":"1698822962000","supportPowConfig":1,"only_device":{"ota":"fail","ota_fail_reason":404},"subOtaInfo":{"state":0,"reason":4,"stateTime":"1698823827000"},"key":0}.

**Comment by bwp91** [comment]
Hi @trhessel 

I just published a beta version of the plugin which should get this device working, let me know how it goes, 

https://github.com/homebridge/homebridge/wiki/How-to-Install-Alternate-Plugin-Versions

**Comment by trhessel** [comment]
Thank you for doing this. I will try this later tonight and sorry to ask
this, but how do I upgrade the plug-in to this code versus finding it in
hoobs and clicking install

**Comment by RykardosJ** [comment]
I installed the beta version of the plugin. The SNZB-01 switch is displayed in the homebridge-UI (it was also displayed before installing the beta version of the plugin) but it is not displayed in the apple homekit application. Any solution?
![Bez názvu](https://github.com/bwp91/homebridge-ewelink/assets/114669256/f8327189-a212-4d00-a461-25a5371e94df)

**Comment by bwp91** [comment]
Are your other ewelink plugin devices appearing in the home app?

**Comment by RykardosJ** [comment]
yes it shows. all other devices are shown only snzb switches are not shown.
